src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int ballsAmount = 3;
+    private static final int NUMBER_COUNT = 100;
+    private final Random randomForNumber;
+    private final ColorSupplier supplier;
+
+    public Lottery() {
+        this.randomForNumber = new Random();
+        this.supplier = new ColorSupplier();
+    }
+
+    public static int getBallsAmount() {
+        return ballsAmount;
+    }

```

Comment:
Let's remove this static method. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-use-static-methods-in-your-solution).